pkg/queue/bounded_queue_test.go

```diff
@@ -30,33 +30,14 @@ import (
 	uatomic "go.uber.org/atomic"
 )
 
-// In this test we run a queue with capacity 1 and a single consumer.
```

Comment:
I would keep the comment at the helper function, it describes more the helper than the actual tests.